src/main/java/Applicaiton.java

```diff
@@ -0,0 +1,214 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import enumeration.Commands;
+
+import static enumeration.Commands.*;
+
+import enumeration.Genre;
+
+import static enumeration.Genre.*;
+
+import models.Song;
+import models.Singer;
+
+/**
+ * The Application class is the main class in the application.
+ */
+public class Applicaiton {
+    private AudioPlayer audioPlayer = new AudioPlayer(this);
+    private Commands currentCommand = PLAY;
+    private Commands previousCommand;
+    private BufferedReader inputReader = new BufferedReader(new InputStreamReader(System.in));
+
+    public static void main(String args[]) throws InterruptedException, IOException {
+
+        Applicaiton app = new Applicaiton();
```

Comment:
This line is not wrong! But we can do better! Let's discuss how we can do the changes!